### PR TITLE
adding ChainLabo saas [Closes #10281]

### LINKED ISSUE #10281: Suggest a staking product or service
### Project name

ChainLabo

### Product type

Staking as a service

### If other product type, please describe

_No response_

### Logo

cl_logo_color

### Description

Using Ethereum.org Launchpad, our customers can create a Non Custodial Solo Staking server with their 32 ETH, and keeping the rewards. ChainLabo manages the updates of the staking nodes which run on dedicated servers hosted in Switzerland

### Website

https://www.chainlabo.com/

### If software is involved, is everything open source?

N/A, no custom software is involved

### Is the project a fork? If yes, which project was forked?

No

### Is the product out of beta development?

N/A

### What wallets support the product or service?

All ethereum wallet

### If the product or service enables staking with <32 ETH, what is the minimum ETH required to stake?

No, minimum 32 ETH and multiples
0% ETH FEES

### If a service, what are the fees associated with using the service?

0% ETH FEES
Startup fee in FIAT (CHF, USD, EUR)
Monthly fee in FIAT (CHF, USD, EUR)

### If the product or service involved a liquidity token, what are the tokens involved?

N/A

### What date did the project or service go live?

01/04/2023

### Has the project undergone an external security audit?

No

### Has the project undergone any security bug bounties?

No

### Is the project being actively maintained?

Yes

### Is the product or service free of trusted/human intermediaries?

During a video call, the customer is guided, without screen sharing, through all the steps for the generation of the keys and the smart contract on the Ethereum.org Launchpad signing. It is customer's responsibility to hold and securely store the keys. ChainLabo only receives the keystore and its corresponding password from the customer.

### If a pooled staking service, can users participate as a node operator without permission?

N/A

### If listing a staking-as-a-service, are users required to sign-up for an account?

NO

### If listing as staking-as-a-service, who has access to the signing keys?

The user and ChainLabo maintain the access to keystore key and password.

Only the user maintain access to 24 words that generate the keystore key

### If a pooled staking service or SaaS provider, what type of withdrawal credentials are being utilized / who holds the withdrawal keys?

Only the user holds  the withdrawal keys.
Withdrawal credentials utilize 0x01 type credentials.
Only the user choose this account

### If a pooled staking service or SaaS provider, what percent of node operators are running a super-majority client?

100% Geth

### If listing node or client tooling, which consensus layer clients (Lighthouse, Teku, Nimbus, or Prysm) are supported?

Nimbus

### What platforms are supported?

Linux

### What user interfaces are supported?

N/A

### Social media links

https://twitter.com/ChainLabo
https://web.telegram.org/a/#-1919442359
https://discord.gg/3xmHwXxk
https://github.com/ChainLabo

### Would you like to work on this issue?

- [ ] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by natpicone**
Adding ChainLabo SaaS provider

relates to https://github.com/ethereum/ethereum-org-website/issues/10281

### CODE REVIEW

File: src/components/icons/staking/ChainLaboGlyphIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from "react"
+import { createIcon } from "@chakra-ui/react"
+import { commonIconDefaultProps } from "../utils"
```
Comment (wackerow) [inline src/components/icons/staking/ChainLaboGlyphIcon.tsx:3]: Can remove this
```suggestion
```

File: src/data/staking-products.json [hunk 2]
```diff
@@ -356,6 +356,39 @@
...
+      "additionalStake": null,
+      "additionalStakeUnit": null,
+      "monthlyFee": 200,
+      "monthlyFeeUnit": "CHF",
+      "isFoss": false,
+      "hasBugBounty": false,
+      "isTrustless": true,
+      "isPermissionless": false,
+      "pctMajorityExecutionClient": 100,
+      "pctMajorityConsensusClient": 100,
```
Comment (wackerow) [inline src/data/staking-products.json:376]: @natpicone You mentioned use of Nimbus, correct?
```suggestion
      "pctMajorityConsensusClient": 0,
```

**Review by wackerow — COMMENTED** [review]
Hey @natpicone! Left a couple comments, otherwise this lgtm!

File: src/data/staking-products.json (hunk 2 above)
Comment (natpicone) [reply src/data/staking-products.json]: Thanks, I missed this

**Comment by natpicone** [comment]
@wackerow can you please review and merge these changes? thx

**Review by wackerow — APPROVED** [review]
Looks good, nice job! Thanks @natpicone

**Comment by wackerow** [comment]
Hey @natpicone, there is a merge conflict since other products have been added to this file since you mad your branch.... I tried to fix it but I think you've blocked any changes to your branch.

Can't merge with those there... Mind fixing the conflicts, or changing the settings so we can edit your branch? thanks!

**Comment by natpicone** [comment]
> Hey @natpicone, there is a merge conflict since other products have been added to this file since you mad your branch.... I tried to fix it but I think you've blocked any changes to your branch.
> 
> Can't merge with those there... Mind fixing the conflicts, or changing the settings so we can edit your branch? thanks!

Hi @wackerow, the branch is updated and the merge conflict is resolved. 
Tests are running, feel free to merge as they finish.
Thanks a lot.

File: docs/ds-implementation.md [hunk 3]
```diff
@@ -17,10 +17,10 @@ Follow the new component directory structure:
...
 src/
 └── components/
-    └── ComponentA/
-          ├── index.tsx
-          ├── ComponentA.stories.tsx
-          └── // Any other files as applicable (utils, child components, useHook, etc.)
+└── ComponentA/
+├── index.tsx
+├── ComponentA.stories.tsx
+└── // Any other files as applicable (utils, child components, useHook, etc.)
```
Comment (wackerow) [inline docs/ds-implementation.md:23]:
```suggestion
    └── ComponentA/
          ├── index.tsx
          ├── ComponentA.stories.tsx
          └── // Any other files as applicable (utils, child components, useHook, etc.)
```

**Pull request merged**